article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+\section{Overview of Causal Graphs}
+\label{sec:graph-overview}
+The previous section, \ref{sec:graph-importance}, motivated why the knowledge encoded in one's DAG is important. 
+This section gives a high level overview of what DAGs are and provides references for more thorough readings on the subject. 
+Causal diagrams and causal graphical models have been introduced by \citet{pearl_causality_1995} as a powerful tool for causal inference, especially in observational studies.
+Perhaps one of the most important and useful features of causal
+graphs when dealing with causal inference problems is the clear illustration
+of the causal relationships between the variables.
+While a formal introduction to the topic of DAGs is beyond the scope of this chapter,
+here we focus specifically on illustrating the power of DAGs to represent
+and encode complex causal relationships between variables in an intuitive and
+clear manner.
+Interested readers can refer to \citet{pearl_causality_2000} for a thorough introduction.
+
+Consider the causal graph represented in Figure \ref{fig:simple-graph}.
+Suppose we're interested in the effect of a treatment Z on Y.
+What the graph in Figure \ref{fig:simple-graph} implies is that Z is
+independent of Y(Z) given X. In causal jargon, the mechanism by which the treatment Z was assigned is ignorable, once we control for covariates X. 
+In such situations, it is sufficient to control for X to obtain an unbiased estimate of the causal effect of Z on Y.
+
+\begin{figure}[h!]
+   \centering
+   \includegraphics[height=0.5\textwidth]{simple-graph.pdf}
+   \caption{Simple DAG of the causal relationships between X, Y and Z}
+   \label{fig:simple-graph}
+\end{figure}
+
+In comparison, note the set of structural equations needed to convey the same assumptions as Figure \ref{fig:simple-graph}.
```

Comment:
In general (and not to be seriously dealt with before @hassan-obeid's quals,) I wonder we are specifically trying to show with this comparison, and I wonder whether this and the following example best show it.

Specifically, what makes it hard or unclear to read the independencies from the structural equations? Assuming you show the equation for $X = f_{X} ( \epsilon_X )$, I can read off the fact that Y does not show up in the equation for Z or its parents, X and epsilon_Z.

Conversely, in the next example, I can read the fact that there is variable U that affects both Z and Y, potentially "explaining away" (i.e. biasing) my results.

I think dags do make our assumptions clearer, but I'm not sure these examples are the best to show that.

cc @bouzaghrane for viz: I think it'd be good for all three of us to come up with 1 or 2 examples from the literature of graphs use to demonstrate the transparency of DAGs vs structural equations. We can then see what the common traits of those dags are and whether we want to reuse any of those examples.

Reply:
Agreed -- will use the language from my prospectus to illustrate this point better. In general, the idea here is that it's possible to encode assumptions in both ways, but one (DAGs) becomes particularly easier to follow as the complexity of the problem arises.

Reply:
I don't really think we need to find real examples tho. The point we're making is intuitive and doesn't need a lot of examples to illustrate it. We just need to change the language so that we're not saying that the advantage of DAGs can be seen in those simple examples. I pushed a commit with suggested changes and tagged you @timothyb0912

Reply:
@hassan-obeid Yes I agree 100%. I have some small nitpicks on the precise language in your edit, but I'll make suggestions on the PR. Thanks for hopping on it swiftly!

Reply:
See edits in #176.